scattr/resources/zona_bb_subcortex/desc-ZonaBB_dseg.tsv

```diff
@@ -13,11 +13,13 @@ index	name
 12	Right GPe
 13	Left GPi
 14	Right GPi
-15	Left Hippocampus
-16	Right Hippocampus
-17	Left NAc
-18	Right NAc
-19	Left Amygdala
-20	Right Amygdala
-21	Left ZI
-22	Right ZI
+15  Left Thalamus
```

Comment:
I think 15 and 16 here should be separated by tabs and aren't

Reply:
Ahh, consequence of vscode. Will fix this.